hypothesis-python/src/hypothesis/extra/array_api.py

```diff
@@ -62,11 +63,23 @@
 ]
 
 
+# Be sure to keep versions in ascending order so api_verson_gt() works
+RELEASED_VERSIONS = ("2021.12",)
+NOMINAL_VERSIONS = RELEASED_VERSIONS + ("draft",)
+NominalVersion = Literal[NOMINAL_VERSIONS]
```

Comment:
I think a `Literal` type has to be actually-literal, so we'll need to write it out twice.  We can then add assertions for consistency:

```python
assert sorted(RELEASED_VERSIONS) == list(RELEASED_VERSIONS)
assert NominalVersion.__args__ == NOMINAL_VERSIONS
```